tsz-compress/Cargo.toml

```diff
@@ -15,8 +15,13 @@ version = { workspace = true }
 crate-type = ["rlib"]
 
 [dependencies]
-bitvec = { version = "1.0.1", default-features = false, features = ["alloc", "atomic"] }
+bitvec = { version = "1.0.1", default-features = false, features = [
+    "alloc",
+    "atomic",
+] }
 tsz-macro = { workspace = true }
+config = "0.13.1"
```

Comment:
I would rather not bring in any more dependencies. We won't end up requiring this one if the compressor has some predefined levels that are runtime callable `Compressor::new(level: TszCompressLevel)`